tikv/cop_handler.go

```diff
@@ -216,6 +216,11 @@ func (svr *Server) buildTableScan(ctx *dagContext, executor *tipb.Executor) (*ta
 		buf:       make([][]byte, len(columns)),
 		loc:       ctx.evalCtx.sc.TimeZone,
 	}
+	if len(columns) == 1 {
+		if columns[0].PkHandle || columns[0].ColumnId == model.ExtraHandleID {
```

Comment:
Do we have a function to check if a column is PK handle in tidb, we should reuse it if we do have.

Reply:
TiDB didn't extract the logic to a function to check the column.

Reply:
Actually, the second check can be removed.